contents/careers/full-stack-software-engineer.md

```diff
@@ -4,6 +4,74 @@ sidebar: Careers
 showTitle: true
 ---
 
-As an early engineer (we currently have a team of 10 in total), you'll be key to building a lot of the core functionality of PostHog. We encourage you to take a look at our repo (issues and PRs) to see the types of things you'd be doing.
+"Let me do it. You tell me when you want it and where you want it to land, and I'll do it backwards and tell you when to take off." - [Katherine Johnson](https://www.nasa.gov/content/katherine-johnson-biography)
 
-We're looking for people who are *proactive* and are *ridiculously quick* at getting new features out, while writing test and generally maintaining a clean codebase.
\ No newline at end of file
+PostHog exists to increase the number of successful products in the world.
+
+## Engineering has been everything to us so far
+
+[James](https://twitter.com/james406) and [Tim](https://twitter.com/timgl) founded PostHog in January. They had a bunch of features they wanted to build to understand their own user behavior - not sharing user data to a third party service, autocapturing events, and getting access to underlying data. Things an engineer would care about.
+
+It wasn't until they met [Dalton Caldwell](https://twitter.com/daltonc?lang=en) for a chat that it all clicked - *open source* would be the strategy, everything else would follow.
+
+This means PostHog is a product-led company. We're building a big open source community around a great, totally free, product.
+
+The end result? Releasing new features regularly has been the *only* thing we've had to do to get thousands of deployments. We've raised millions of dollars from the world's best investors but we're still a small, lean team.
+
+PostHog helps improve products built by teams in everything from huge airlines and banks, to indie gamers making it more fun to protect earth from aliens and even underwear startups working on their retention!
```

Comment:
Phrasing could be better to make it easier to read. I think the correct form here is `from w to x to y to z`